File: represent/bench/java_flat/SpecGenBench_0300__SpecGenBench__common__FindFirstZero__FindFirstZero.java
public class FindFirstZero {

    	 public static int FindFirstZero(int[] x) {
         	assert x.length >= 0;
         	if (x.length == 0) {
            		return -1;
        	} else {
            		int index = 0;

            		while (x.length - index > 0 && x[index] != 0) {
                		index = index + 1;
            		}
            		if (x.length - index == 0) {
               			index = -1;
            		}
            	return index;
        	}
    	}
}

File: represent/bench/c_flat/SpecGenBench_0300__SpecGenBench__common__FindFirstZero__FindFirstZero.c
int FindFirstZero(int * x, int x_len);

int FindFirstZero(int * x, int x_len) {

         	assert x_len >= 0;
         	if (x_len == 0) {
            		return -1;
        	} else {
            		int index = 0;

            		while (x_len - index > 0 && x[index] != 0) {
                		index = index + 1;
            		}
            		if (x_len - index == 0) {
               			index = -1;
            		}
            	return index;
        	}
}
